typescript/src/accessors/internal_accessor.ts

```diff
@@ -127,6 +127,10 @@ export class InternalAccessor implements IConsumerApiAccessor {
     return this._consumerApiService.getProcessModelById(identity, processModelId);
   }
 
+  public async getProcessModelByProcessInstanceId(identity: IIdentity, processInstanceId: string): Promise<DataModels.ProcessModels.ProcessModel> {
+    return this._consumerApiService.getProcessModelByProcessInstanceId(identity, processInstanceId);
```

Comment:
Isn't there an `await` missing? 

```suggestion
    return await this._consumerApiService.getProcessModelByProcessInstanceId(identity, processInstanceId);
```

Reply:
The `await` would not do anything in this case.